mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+                  properties:
+                    - !ruby/object:Api::Type::Enum
+                      name: 'statistic'
+                      values:
+                        - :STATISTIC_UNDEFINED
+                        - :MEAN
+                        - :MIN
+                        - :MAX
+                    - !ruby/object:Api::Type::String
+                      name: 'minValue'
+                      description: |
+                        The minimum column statistic value allowed for a row to pass this validation.
+                        At least one of minValue and maxValue need to be provided.
+                    - !ruby/object:Api::Type::String
+                      name: maxValue
+                      description: |
+                        The maximum column statistic value allowed for a row to pass this validation.
+                        At least one of minValue and maxValue need to be provided.
+                    - !ruby/object:Api::Type::Boolean
+                      name: 'strictMinEnabled'
+                      default_value: false
+                      description: |
+                        Whether column statistic needs to be strictly greater than ('>') the minimum, or if equality is allowed.
+                        Only relevant if a minValue has been defined. Default = false.
+                    - !ruby/object:Api::Type::Boolean
+                      name: 'strictMaxEnabled'
+                      default_value: false
+                      description: |
+                        Whether column statistic needs to be strictly lesser than ('<') the maximum, or if equality is allowed.
+                        Only relevant if a maxValue has been defined. Default = false.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'rowConditionExpectation'
+                  description: |
+                    Table rule which evaluates whether each row passes the specified condition.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'sqlExpression'
+                      description: |
+                        The SQL expression.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'tableConditionExpectation'
+                  description: |
+                    Table rule which evaluates whether the provided expression is true.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'sqlExpression'
+                      description: |
+                        The SQL expression.
+            - !ruby/object:Api::Type::Boolean
+              name: 'passed'
+              description: |
+                Whether the rule passed or failed.
+            - !ruby/object:Api::Type::String
+              name: 'evaluatedCount'
+              default_values: 0
```

Comment:
if this is output, we shouldn't have a default here

Reply:
+1